forms/modelutils.py

```diff
@@ -103,6 +103,11 @@
     ('N', _('(2) No')),
 )
 
+MONITOR_MODE = (
+    ('SHORT', _('Short')),
+    ('LONG', _('Long')),
```

Comment:
```suggestion
    (1, _('Long')),
    (2, _('Short')),
```

1. Since we expect some sort of processing in external systems like excel, etc., it's best we stick with numbers.. or at least single char., and
2. I _think_ **Long** should be the priority i.e. what appears first.